packages/atomic/src/components/commerce/atomic-commerce-pager/atomic-commerce-pager.new.stories.tsx

```diff
@@ -1,37 +1,62 @@
-import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import type {Meta, StoryObj as Story} from '@storybook/web-components-vite';
+import {getStorybookHelpers} from '@wc-toolkit/storybook-helpers';
 import {wrapInCommerceInterface} from '@/storybook-utils/commerce/commerce-interface-wrapper';
 import {parameters} from '@/storybook-utils/common/common-meta-parameters';
-import {renderComponent} from '@/storybook-utils/common/render-component';
 
-const {decorator, play} = wrapInCommerceInterface();
+const {decorator, afterEach} = wrapInCommerceInterface();
+const {events, args, argTypes, template} = getStorybookHelpers(
+  'atomic-commerce-pager',
+  {excludeCategories: ['methods']}
+);
 
 const meta: Meta = {
   component: 'atomic-commerce-pager',
   title: 'Commerce/Pager',
   id: 'atomic-commerce-pager',
-  render: renderComponent,
+  render: (args) => template(args),
   decorators: [decorator],
-  parameters,
-  play,
+  parameters: {
+    ...parameters,
+    actions: {
+      handles: events,
+    },
+  },
+  args: {
+    ...args,
+    'number-of-pages': '5',
+  },
```

Comment:
Not needed from what I tested
```suggestion
  args,
```